README.md

```diff
@@ -37,6 +37,7 @@ of riscv-tools:
     $ cd rocket-chip/riscv-tools
     $ git submodule update --init --recursive
     $ export RISCV=/path/to/install/riscv/toolchain
+    $ export MAKEFLAGS="$MAKEFLAGS -jN"
```

Comment:
To make this user-proof, put a number here instead of N.  If someone follows these directions literally, they'll get an error when they try to run make.

    export MAKEFLAGS="$MAKEFLAGS -j16"